HISTORY.md

```diff
@@ -7,6 +7,7 @@
 * `DBOptions::bytes_per_sync` and `DBOptions::wal_bytes_per_sync` can now be changed dynamically, `DBOptions::wal_bytes_per_sync` will flush all memtables and switch to a new WAL file.
 * Support dynamic adjustment of rate limit according to demand for background I/O. It can be enabled by passing `true` to the `auto_tuned` parameter in `NewGenericRateLimiter()`. The value passed as `rate_bytes_per_sec` will still be respected as an upper-bound.
 * Support dynamically changing `ColumnFamilyOptions::compaction_options_fifo`.
+* Add a new db properties "rocksdb.estimated-oldest-data-time" to return oldest data timestamp. Currently the property is available only for FIFO compaction with compaction_options_fifo.allow_compaction = false.
```

Comment:
The property name that you added in db.h is called estimated-earliest-key-timestamp.
Lets rename it everywhere to be called oldest-key-time, to be consistent.